tests/hawc/services/nih/test_pubmed.py

```diff
@@ -0,0 +1,206 @@
+import os
+from unittest import TestCase
+
+import pytest
+
+from hawc.services.nih import pubmed
+
+PUBMED_API_KEY = os.environ.get("PUBMED_API_KEY")
+
+
+@pytest.mark.skipif(
+    PUBMED_API_KEY is None, reason="Test environment requires $PUBMED_API_KEY to be set."
+)
+class PubMedSearchTests(TestCase):
```

Comment:
drop the TestCase parent class if you rename to `TestPubMed` and do things the "pytest way"